pkg/scheduler/scheduler.go

```diff
@@ -338,7 +337,7 @@ func initPolicyFromConfigMap(client clientset.Interface, policyRef *kubeschedule
 }
 
 // NewFromConfig returns a new scheduler using the provided Config.
-func NewFromConfig(config *factory.Config) *Scheduler {
+func NewFromConfig(config *Config) *Scheduler {
```

Comment:
can this be unexported already?

Reply:
the only blocker now is the call in cluster-autoscaler.

Reply:
Since they are locked to an old version, and we already have an alternative for when they decide to upgrade, I would say this is not a blocker.

Reply:
Okay, could be addressed in a separate PR. WDYT?

Reply:
sg